src/hooks/useTokenAmountToUsd.js

```diff
@@ -0,0 +1,92 @@
+import { useEffect, useState } from 'react'
+import { captureException } from '@sentry/browser'
+
+import { useCourtConfig } from '../providers/CourtConfig'
+import { useUniswapAnjPrice } from './useUniswapAnjPrice'
+import { bigNum, formatUnits } from '../lib/math-utils'
+import { getNetworkType } from '../lib/web3-utils'
+
+const API_BASE = 'https://api.0x.org'
+const PRECISION = bigNum(10).pow(18)
```

Comment:
Just checking, since USDC only has 6 decimals, is the API precision always 18 units, or is it dependent on the token to sell?

Reply:
Ah yeah, they actually come up in `sellToken` denomination. Good catch